ubuntu.json

```diff
@@ -328,5 +328,10 @@
     "alias": "ubuntu-toolchain-r-test",
     "sourceline": "ppa:ubuntu-toolchain-r/test",
     "key_url": null
+  },
+  {
+    "alias": "yarn",
+    "sourceline": "deb http://dl.yarnpkg.com/debian/ stable main",
+    "key_url": "http://dl.yarnpkg.com/debian/pubkey.gpg"
```

Comment:
This is also available over HTTPS, if your system handles loading GPG keys via HTTPS 😄